### ISSUE TITLE
Installation seems to require "tqdm"

### ISSUE THREAD

**Issue by vmullig** [danielegrattarola/spektral#123]
I'm finding that when I `pip install spektral`, my scripts fail with the following error:

```
  File "python/train_pnear_est_v5_10mer_E.py", line 74, in <module>
    from spektral.layers import GCNConv, GlobalSumPool
  File "/mnt/sdceph/users/vmulligan/Tensorflow2/venv/lib64/python3.6/site-packages/spektral/__init__.py", line 2, in <module>
    from . import datasets
  File "/mnt/sdceph/users/vmulligan/Tensorflow2/venv/lib64/python3.6/site-packages/spektral/datasets/__init__.py", line 4, in <module>
    from .modelnet import ModelNet
  File "/mnt/sdceph/users/vmulligan/Tensorflow2/venv/lib64/python3.6/site-packages/spektral/datasets/modelnet.py", line 9, in <module>
    from tqdm import tqdm
ModuleNotFoundError: No module named 'tqdm'
```

The workaround is to first `pip install tqdm`.  This is obviously a minor issue, but it would be good if the original pip install of Spektral found and installed all of the dependencies.

**Comment by danielegrattarola** [comment]
Thanks, I just released a fix (v1.0.1)!

**Comment by vmullig** [comment]
>Thanks, I just released a fix (v1.0.1)!

That was super fast!  Thanks, @danielegrattarola!
